docs/_components/spinner.md

```diff
@@ -13,7 +13,7 @@ Spinners can be helpful to signal to a user that the current process may require
 {% capture example %}
 <div class="lg-spinner">
   <div>
-    <img src="{{ site.baseurl }}/assets/img/spinner.gif" srcset="{{ site.baseurl }}/assets/img/spinner@2x.gif" width="144" height="144" class="text-middle" alt="Spinner" />
+    <img src="{{ site.baseurl }}/assets/img/spinner.gif" srcset="{{ site.baseurl }}/assets/img/spinner.gif, {{ site.baseurl }}/assets/img/spinner@2x.gif 2x" width="144" height="144" class="text-middle" alt="Spinner" />
```

Comment:
maybe easier to read on multiple lines?

```html
    <img src="{{ site.baseurl }}/assets/img/spinner.gif"
         srcset="{{ site.baseurl }}/assets/img/spinner.gif,
                 {{ site.baseurl }}/assets/img/spinner@2x.gif 2x"
         width="144"
         height="144"
         class="text-middle"
         alt="Spinner" />
```

Reply:
> maybe easier to read on multiple lines?

Yep, I agree. It definitely reads better. One possible point of confusion is how to align the subsequent lines of the attribute value. In this case, it seems straightforward enough to use spaces to vertically align it with the start of the previous line. If a project uses tabs instead of spaces, it might not be so clear. I think in that case, I might simply recommend to tab one extra level than the preceding line.